### DISCUSSION TITLE
Vercel "Out of Memory" ("OOM") error

### DISCUSSION THREAD

**Discussion by stephenasuncionDEV** [vercel/next.js#58713] [Help]
### Summary

I was trying to build a preview env for one of my website but I'm getting this out of memory error.

![image](https://github.com/vercel/next.js/assets/73958774/70a17c6f-483c-45e7-862d-756f37d7e413)

### Additional information

I tried switching to pnmp, only got 4 errors now:
![image](https://github.com/vercel/next.js/assets/73958774/2c3d7d41-48b1-45ab-a032-dfe7a93d0a11)

**Comment by icyJoseph** [comment]
Mmm I'd also open a ticket with Vercel Support. If we take the logs at face value, it'd look like the SWC binary is too big for Vercel?  Open a ticket with them, provide your project name, and logs you get.

**Reply by stephenasuncionDEV** [reply]
Will do thanks!

**Comment by MananTank** [comment]
@stephenasuncionDEV Did you get any solution for this from Vercel team? I'm having the same issue
